src/applications/edu-benefits/10216/sass/10216-edu-benefits.scss

```
@@ -0,0 +1,7 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-process-list";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-form-process";
+@import "../../../../platform/forms/sass/m-schemaform";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
```

I know this is boilerplate, but it would be immensely helpful to the design system team if you updated these import paths to point to css-library instead. We're actively deprecating Formation and we probably need to look at the new app script and update whatever spits this boilerplate out. 
```suggestion
@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-process-list";
@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-form-process";
@import "../../../../platform/forms/sass/m-schemaform";
@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-modal";
@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-omb-info";
```

---

This has been updated; yeoman generator will need updates for these files, va buttons, to fix `OMBInfo` to be `va-omb-info`, to update reference of vagov-content repo to content-build repo, etc. It would be fantastic if the generator script was updated; our team will be digitizing another 12 forms